metadata-ingestion/src/datahub/ingestion/source/sql/bigquery.py

```diff
@@ -431,9 +444,12 @@ def _get_exported_bigquery_audit_metadata(
         start_time: str = (
             self.config.start_time - self.config.max_query_duration
         ).strftime(BQ_DATETIME_FORMAT)
+        self.report.start_time = start_time
```

Comment:
Why is this being set here again? Don't we get it already from lines 403 & 408?

Reply:
will fix to rename these properly to avoid confusion